TRACK_README.md

```diff
@@ -16,57 +16,52 @@ Please read about how to [get involved in a track](https://github.com/exercism/d
 
-Please keep the following in mind:
+If you plan to make significant or breaking changes, please open an issue so we can discuss it first. If this is a discussion that is relevant to more than just the {{LANGUAGE}} track, please open an issue in [exercism/discussions](https://github.com/exercism/discussions/issues).
 
-- Pull requests should be focused on a single exercise, issue, or change.
+## Submitting a Pull Request
 
-- We welcome changes to code style, and wording. Please open a separate PR for these changes if possible.
+Pull requests should be focused on a single exercise, issue, or conceptually cohesive change. Please refer to Exercism's [pull request guidelines](https://github.com/exercism/docs/blob/master/contributing/pull-request-guidelines.md).
 
-- Please open an issue before creating a PR that makes significant (breaking) changes to an existing exercise or makes changes across many exercises. It is best to discuss these changes before doing the work. Discussions related to exercises that are not track specific can be found in [exercism/discussions](https://github.com/exercism/discussions/issues).
+Please follow the coding standards for {{LANGUAGE}}. (If there is a formatter for the track's language, add instructions for using it here.)
 
-- Follow the coding standards for {{LANGUAGE}}. (If there is a formatter for the track's language, add instructions for using it here.)
+### Verifying Your Change
 
-- Watch out for trailing spaces, extra blank lines, and spaces in blank lines.
+Before submitting your pull request, you'll want to verify the changes in two ways:
 
-- All the tests for {{LANGUAGE}} exercises can be run from the top level of the repo with ... Please run this command before submitting your PR.
+* Run all the tests for the {{LANGUAGE}} exercises
+* Run an Exercism-specific linter to verify the track
 
-## Contributing a New Exercise ##
+All the tests for {{LANGUAGE}} exercises can be run from the top level of the repo with
 
-- All Exercism exercises must be defined in [problem-specifications](https://github.com/exercism/problem-specifications/tree/master/exercises) before they are implemented for a specific track. Please submit a PR there if your exercise is new to Exercism.
+```
+# add this command
+```
 
-- Please make sure the new exercise conforms to specifications in the [exercism/problem-specifications](https://github.com/exercism/problem-specifications) repo.
+For the Exercism-specific linting, please see [the documentation](https://github.com/exercism/docs/blob/master/language-tracks/configuration/linting.md).
 
-- Each exercise must stand on its own. Do not reference files outside the exercise directory. They will not be included when the user fetches the exercise.
+## Contributing a New Exercise
 
-- Exercises should use only the {{LANGUAGE}} core libraries.
+Please see the documentation about [adding new exercises](https://github.com/exercism/docs/blob/master/you-can-help/make-up-new-exercises.md).
```

Comment:
OK, I've thought about this for a bit. I think my solution is going to be to rename the "port an exercise" guide to be "implement an exercise" (implied: given a problem specification, either general or custom).

I'll do this in a separate set of PRs, since I'll need to rename references to it across all of Exercism.